catalog-entities/marketplace/plugins/scaffolder-relation-processor-catalog-integration.yaml

```diff
@@ -0,0 +1,54 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/redhat-developer/rhdh-plugins/refs/heads/main/workspaces/marketplace/json-schema/plugins.json
+apiVersion: extensions.backstage.io/v1alpha1
+kind: Plugin
+metadata:
+  name: scaffolder-relation-processor-catalog-integration
```

Comment:
Can you use this plugin name in the package "  name: backstage-community-plugin-catalog-backend-module-scaffolder-re"? I guess it is corresponding package...